buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,18 @@
+plugins {
+    `kotlin-dsl`
+
+}
+repositories {
+    mavenCentral()
+    jcenter()
+    maven("https://files.minecraftforge.net/maven")
+
+}
+
+dependencies {
+    compileOnly("net.minecraftforge.gradle:ForgeGradle:2.3-SNAPSHOT")
```

Comment:
Is it possible to move all ForgeGradle related code to SpongeCommon? SpongeAPI does not rely on proprietary Minecraft code and we should try to keep it that way for other implementations (e.g. Lantern).